linkml_runtime/utils/schemaview.py

```diff
@@ -108,25 +108,6 @@ def is_absolute_path(path: str) -> bool:
     drive, tail = os.path.splitdrive(norm_path)
     return bool(drive and tail)
 
-def _resolve_import(source_sch: str, imported_sch: str) -> str:
-    if os.path.isabs(imported_sch):
-        # Absolute import paths are not modified
-        return imported_sch
-    if urlparse(imported_sch).scheme:
-        # File with URL schemes are not modified
-        return imported_sch
-    
-    if WINDOWS:
-        path = PurePath(os.path.normpath(PurePath(source_sch).parent / imported_sch)).as_posix()
-    else:
-        path = os.path.normpath(str(Path(source_sch).parent / imported_sch))
-
-    if imported_sch.startswith(".") and not path.startswith("."):
-        # Above condition handles cases where both source schema and imported schema are relative paths: these should remain relative
-        return f"./{path}"
-
-    return path
```

Comment:
summary of logic in this function:

- if the `imported_sch` path is absolute or is URI / CURIE-like, leave as-is
- normalise the `imported_sch` path, assuming that `source_sch` and `imported_sch` are siblings
- add `./` to the normalised `imported_sch` path if it originally started with `./` (not really necessary)

==> this can be refactored and simplified